code-server/run.sh

```diff
@@ -47,4 +47,4 @@ fi
 
 echo "👷 Running code-server in the background..."
 echo "Check logs at ${LOG_PATH}!"
-$CODE_SERVER --auth none --port ${PORT} > ${LOG_PATH} 2>&1 &
+$CODE_SERVER --auth none --port ${PORT} --app-name ${APP_NAME} > ${LOG_PATH} 2>&1 &
```

Comment:
I am not sure if we sanitize variables but I wonder if we should add quotes in case the name contains spaces.  Hopefully no one puts quotes in their name, but maybe we also want to escape quotes that might be in the app name just in case or add sanitization if we do not already have it.

```suggestion
$CODE_SERVER --auth none --port ${PORT} --app-name "${APP_NAME}" > ${LOG_PATH} 2>&1 &
```

Reply:
Thanks for the suggestions! So I added the quotes, but for now did not escaped the input. As you said, we can later deal with this, if it is becomes an issue.